roles/cis_security/tasks/type-files/redhat-7-type.yml

```diff
@@ -1609,7 +1609,7 @@
         replace: "{{ item.replace }}"
       loop:
         - {find: '^max_log_file\s+=\s+[^{{ log_file_size }}]', replace: 'max_log_file = {{ log_file_size }}'} # 4.1.1.1
-        - {find: '^max_log_file_action\s+=\s+((?!{{ max_log_file_action }}).)*$', replace: 'max_log_file_action ={{ max_log_file_action }} '} # 4.1.1.2
+        - {find: '^max_log_file_action\s+=\s+(.*)$', replace: 'max_log_file_action ={{ max_log_file_action }} '} # 4.1.1.2
```

Comment:
This will case this to always run not just when it doesn't match

Reply:
got it reverting